happypose_examples/config/cosypose_params.yaml

```diff
@@ -34,6 +36,25 @@
           # Labels of detected objects to keep. If not specyfied, all objects are kept
           # labels_to_keep:
           #   - "ycbv-obj_000001"
+        icp:
+          # Minimum number of matching depth points to consider detection valid
+          n_min_points: 1000
+          # Minimum clipping distance of depth point from the camera to be considered valid
+          min_measured_depth: 0.2
+          # Maximum clipping distance of depth point from the camera to be considered valid
+          max_measured_depth: 5.0
+          # Number of iterations of ICP algorithm
+          iterations: 100
+          # Controls the accuracy of registration at each iteration of ICP
+          tolerance: 0.05
```

Comment:
Can you be more precise?
How is the accuracy evaluated? What unit?

Reply:
I literally copied it from OpenCV documentation

Reply:
So at least say something like: *this is the tolerance passed to this OpenCV function*.

Reply:
After looking at the source code of `cv::ppf_match_3d::ICP`, some comments that could be added to this doc:

1. tolerance is a percentage. It tells the algo to stop when the norm of the error vector makes less improvement in percent than this threshold. The error function is the norm of the difference of the matched 3D points.
2. num_levels corresponds to iterative ICP where the original point cloud is subsampled. Note that if the output of CosyPose is considered good, num_levels does not have to be high.

See [this function](https://github.com/opencv/opencv_contrib/blob/ce3c6681c9bf0e5cf46704a6ce0883078bdba074/modules/surface_matching/src/icp.cpp#L246)

Reply:
Updated in https://github.com/agimus-project/happypose_ros/pull/23/commits/4f7402f10b817e69dd746cb1bb61d403296107de